js/classes/entities/midground/rain.js

```diff
@@ -0,0 +1,97 @@
+class Rain {
+    constructor(dir, pos) {
+        // TODO use direction to rotate raindrop and choose different travel path.
+        this.dir = dir;
+        this.origin = pos;
+        this.pos = pos;
+        this.velocity = new Vector(0, 0);
+        Rain.spriteSheet = Rain.SPRITESHEET_DOWN;
+        if (dir == "down") {
+            Rain.spriteSheet = Rain.SPRITESHEET_DOWN;
+        } else if (dir == "left") {
+            Rain.spriteSheet = Rain.SPRITESHEET_LEFT;
+        } else if (dir == "right") {
+            Rain.spriteSheet = Rain.SPRITESHEET_RIGHT;
+        }
+        this.animator = new Animator(Rain.spriteSheet, new Vector(0, 0), new Vector(Rain.SIZE.x, Rain.SIZE.y), 1, 1);
+        this.hasStartedRaining = false;
+        this.stopRain = false;
+    }
+    static get SPRITESHEET_DOWN() {
+        return "./sprites/rain_drop.png";
+
+    }
+    static get SPRITESHEET_LEFT() {
+        return "./sprites/rain_drop_left.png";
+
+    }
+    static get SPRITESHEET_RIGHT() {
+        return "./sprites/rain_drop_right.png";
+
+    }
```

Comment:
It may be good to condense the different rain direction spritesheets onto a single rain spritesheet.